src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -149,7 +149,7 @@ public override bool IsSealed
 
         public override bool IsAbstract
```

Comment:
There are many single-line properties and methods in this file, including directly above and below this property. Should I change those too?